src/applications/personalization/components/NameTag.jsx

```diff
@@ -71,7 +71,7 @@ const NameTag = ({
   userFullName: { first, middle, last, suffix },
   latestBranchOfService,
   showBadgeImage,
-  showUpdatedNameTag,
+  showDisabilityRating,
```

Comment:
I'm pretty sure we don't need this prop at all anymore. I'll double check.

Reply:
Yeah, we actually don't need this. 🎉 Basically everything related to the old `showUpdatedNameTag` should be removed and all code should act as though that value is `true`.